### PR TITLE
Adding PWA to theme edition

### PR DESCRIPTION
**Pull request by dobooth**
This PR adds PWA to the edition list. Adds color and svg for PWA for PWA-only content.

### CODE REVIEW

**Comment by dobooth** [comment]
I suspect one of those js files should be removed.

**Comment by dshevtsov** [comment]
Opened a PR to fix the linter's issue: https://github.com/magento-devdocs/devdocs-theme/pull/99

**Comment by dshevtsov** [comment]
@dobooth please use more descriptive messages for commits; _Initial commit_ is very unclear.

**Comment by dshevtsov** [comment]
The title is also unclear.

**Review by bdenham — APPROVED** [review]
@dobooth See updates I made to your branch that shows some of the processes for developing and updating the theme:

1. Updated the faux version number in the `.gemspec` file and the header and footer files. This helps invalidate the caching on the server. Otherwise, it could take some time for the server to invalidate and update the cached pages.

2. Changed the PWA icon and Label colors to Spectrum blue and b2b to Spectrum green. I did this because Labels in Spectrum have been deprecated, replaced by Badges, I think. So to prevent a lot of rework, I went with the old deprecated `spectrum-Label--blue` for PWA and changed B2B to `spectrum-Label--green` because the preview `seafoam` color was too close to blue. :)

3. Reduced the size of the indicator circles. They were a bit too big.

4. Refactored the SVGs to make smaller.

5. Added the new pwa-only icon styles to some files in the `example` directory to test them locally. If you run `npm run watch:all` and use the example files to test, you can get automatic updates as you make changes to the scss files. Very handy.

That's it for the theme side. Once we merge this, we can go over how to test and update the consumers of the newly updated theme: devdocs, merchdocs, etc.

File: _includes/layout/edition-callout.html [hunk 1]
```diff
@@ -22,6 +22,14 @@
   </span>
 </div>
 {% endif %}
+{% if page.pwa_only == true %}
+<div class="break">
+  <span class="edition-Label pwa-only">{{site.data.var.pwa}} only.
+    <a class="spectrum-Link" href="https://docs.magento.com/user-guide/getting-started.html#product-editions">
```
Comment (keharper) [inline _includes/layout/edition-callout.html:28]: Consider getting rid of the link. I doubt Merchdocs/ExL would have PWA features.

**Review by keharper — COMMENTED** [review]
The B2B image appears on merchdocs, so you'll need to update it there too.

**Review by bdenham — APPROVED** [review]

**Comment by dshevtsov** [comment]
Please update internal docs and announce the new feature in a team's Slack.

**Pull request merged**